msal/samples/desktop/SampleApp/MainForm.cs

```diff
@@ -43,7 +43,7 @@ public MainForm()
             tabControl1.Appearance = TabAppearance.FlatButtons;
             tabControl1.ItemSize = new Size(0, 1);
             tabControl1.SizeMode = TabSizeMode.Fixed;
-            user = _msalHelper.Application.Users.First();
+            user = _msalHelper.Application.GetUsers().Result.First();
```

Comment:
>First [](start = 61, length = 5)

FirstOrDefault ? and make the sample app handle null elegantly?

Reply:
changed